build.gradle.kts

```
@@ -1,37 +1,27 @@
-@file:Suppress("UnstableApiUsage")
-
 // Plugins
 plugins {
-	signing
+	`signing`
```

no need backtick quoting that
```suggestion
	signing
```